db/migrate/20200210235313_drop_service_provider_requests.rb

```diff
@@ -0,0 +1,5 @@
+class DropServiceProviderRequests < ActiveRecord::Migration[5.1]
+  def change
+    drop_table :service_provider_requests
```

Comment:
The old code still reads this table as a fall-through right? It may try to establish a connection to the table....which would go very poorly if dropped WDYT of moving this drop into a 3rd PR?

Reply:
Correct.  I'm dropping it now to prove it will function without the table (and catch any problems) but the actual drop will be in a 3rd pr similar to what I did with auth apps https://github.com/18F/identity-idp/pull/3524

Reply:
This was marked as resolved but changes are still here -- just checking the plan is to not change the schema in this PR right?